src/main/java/com/easyrest/pages/OwnerPanel.java

```diff
@@ -38,6 +34,18 @@ public OwnerPanel(WebDriver driver) {
         waiters = new Waiters(driver);
     }
 
+    private boolean isDisplayedElement(By element) {
+        return this.driver.findElement(element).isDisplayed();
+    }
+
+    public boolean isEmployeeDisplayed(String name) {
+        return isDisplayedElement(By.xpath("*//div//span[normalize-space()='" + name + "']"));
```

Comment:
why did you write x path here and not where is variable?

Reply:
I need to substitute a name from a method in xpath.
That's why I can not write is in variable.